### PR TITLE
Migration/textdynamics

### PR DESCRIPTION
**Pull request by rvilarl**
TextDynamics!

### CODE REVIEW

File: src/textdynamics.ts [hunk 1]
```diff
@@ -98,17 +110,17 @@ export class TextDynamics extends Note {
   }
 
   // Draw the dynamics text on the rendering context
-  draw() {
+  draw(): void {
     this.setRendered();
     const x = this.getAbsoluteX();
-    const y = this.stave.getYForLine(this.line + -3);
+    const y = check<Stave>(this.stave).getYForLine(this.line + -3);
```
Comment (0xfe) [inline src/textdynamics.ts:116]: Why not `this.checkStave()`?

Comment (rvilarl) [reply src/textdynamics.ts:116]: Changed

**Comment by rvilarl** [comment]
Ready to merge

**Comment by 0xfe** [comment]
lgtm

**Pull request merged**